lib/cc/engine/analyzers/violation.rb

```diff
@@ -45,7 +64,9 @@ def sorted_hashes
         end
 
         def other_sexps
-          @other_locations ||= sorted_hashes.drop(1)
+          sorted = sorted_hashes
```

Comment:
I definitely agree that the class seems wonky. Just wasn't sure if fixing it was in the scope of this PR. 

Added an `issue` class - corresponding to occurrence of duplication code - to address this problem.